crates/nu-cmd-lang/src/core_commands/error_make.rs

```diff
@@ -138,7 +138,7 @@ fn make_error(value: &Value, throw_span: Option<Span>) -> Option<ShellError> {
                         if start > end {
                             Some(ShellError::GenericError(
                                 "invalid error format.".into(),
-                                "`$.label.start` is stricly bigger than `$.label.end`".into(),
+                                "`$.label.start` is strictly bigger than `$.label.end`".into(),
```

Comment:
I would rephrase it in terms of how it shold be but that is a stylistic choice

Reply:
ooh gotcha :ok_hand:

Reply:
i changed it in 806158fe9a41402a3e9e19a491e4c622b63c6219 :wink: